src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,139 @@
+package duke;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * The class that deals with loading tasks from the file and saving tasks in the file.
+ */
+class Storage {
+
+    /**
+     * Creates a new text file so that the list of tasks can be saved on the hard disk.
+     */
+    public static void createFile() throws java.io.IOException {
```

Comment:
I am not sure why you are using `java.io.IOException` instead of just `IOException`.